package.json

```diff
@@ -21,7 +21,7 @@
   "main": "./dist/electron/main.js",
   "scripts": {
     "compile:electron": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.electron.config.js",
-    "compile:web": "yarn copyenv && cd web && node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.config.js",
+    "compile:web": "yarn copyenv && cd web && NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.config.js",
```

Comment:
This change forces `dev:web-server` to always be in `production`.  We still need it to be in `development`.

For `comple:web`, I believe `NODE_ENV` is meant to be given at the command line when it is called.  It will be set to production at the build machine script.

Reply:
Ah, good call. In the CI environment, the variable wasn't getting propagated after these changes.  Since this was done over a stretched out time, I'd forgotten that this was ultimately run under `dev:web`